test/TestAssets/SimpleTestProject3/TestSessionTimeoutTest.cs

```diff
@@ -8,19 +8,19 @@ namespace SampleUnitTestProject3
     public class TestSessionTimeoutTest
     {
         [TestMethod]
-        public void TestWhichTakeSomeTime1()
+        public void TestWhichTakeSomeTime11()
```

Comment:
Why did we need this change ?

Reply:
This class has 6 unittest each with 3 seconds sleep. I wanted to use three tests only so appended 1 in three test to use in TestCasefilter. But this is not usefull now as I am removing Acceptance test and adding UT for that.